endpoint/BUILD

```diff
@@ -3,8 +3,8 @@ cc_library(
     srcs = ["endpoint.c"],
     hdrs = ["endpoint.h"],
     defines = [
-        "ENDPOINT_HOST=\"tangle-accel.biilabs.io\"",
-        "ENDPOINT_PORT=\"443\"",
+        "ENDPOINT_HOST=\"localhost\"",
+        "ENDPOINT_PORT=\"8000\"",
```

Comment:
Should we keep in `443` port? We are using HTTPS only in `endpoint`. I think `443` would make more sense